src/components/search_component/SearchComponent.js

```diff
@@ -10,17 +10,24 @@ export const SearchComponent = ({ handleChange }) => {
   };
 
   return (
-    <div className="searchBar">
+    <div className="searchContainer">
```

Comment:
With JSX class names, I like to do JSX class names with kebab case ( example: `"search-container"` ) so I can differentiate them from JS